core/datasource/spec/spec.go

```diff
@@ -71,6 +71,13 @@ func New(originalSpec datasource.Spec) (*Spec, error) {
 		}
 	}
 
+	builtInTrigger := false
+	for _, f := range filtersFromSpec {
+		if strings.HasPrefix(f.Key.Name, "vegaprotocol.timetrigger") && f.Key.Type == datapb.PropertyKey_TYPE_TIMESTAMP {
```

Comment:
Is that correct or should it be vegaprotocol.builtin.timetrigger ?

Reply:
Yes, this is incorrect, it should be ` vegaprotocol.builtin.timetrigger `, missed that. 
Thanks

Reply:
Corrected now